Makefile

```diff
@@ -18,7 +18,7 @@ replaceloadingpage:
 
 # Extract the whl file
 extractkolibriwhl:
-	unzip -q "src/kolibri*.whl" "kolibri/*" -d src/
+	unzip -q "src/kolibri*.whl" "kolibri/*" -x /kolibri/dist/cext/ -d src/
```

Comment:
Did you test this?

Reply:
```

Reply:
```

Reply:
```

Reply:
```
lucye:/data/data/org.le.kolibri/files/kolibri/dist $ ls                        
CherryPy-6.2.0.post20171012-py2.7.egg-info le_utils                     
CherryPy-6.2.0.post20171016-py2.7.egg-info metaphone                    
CherryPy-6.2.0.post20171017-py2.7.egg-info morango                      
README                                     morango-0.0.2-py2.7.egg-info 
__init__.py                                mptt                         
args.py                                    porter2stemmer               
cext                                       pyasn1                       
cherrypy                                   pytz                         
clint                                      requests                     
colorlog                                   requests_toolbelt            
concurrent                                 rest_framework               
dateutil                                   rest_framework_csv           
django                                     rsa                          
django_filters                             six.py                       
django_js_reverse                          sqlalchemy                   
docopt.py                                  tests                        
iceqube                                    tqdm                         
ifcfg                                      tzlocal                      
jsonfield                                  unicodecsv                   
kolibri_exercise_perseus_plugin         
```

Reply:
`unzip -q "src/kolibri*.whl" "kolibri/*" -x */cext/* -d src/`

Reply:
I did test this, and I didn't see it unzipped o.O 

Excluding a directory should be automatically recursive, per the `unzip` docs. I'll look into it